pandas/core/indexes/datetimes.py

```diff
@@ -1727,7 +1727,32 @@ def freq(self, value):
     is_month_start = _field_accessor(
         'is_month_start',
         'is_month_start',
-        "Logical indicating if first day of month (defined by frequency)")
+        """
+        Return a boolean indicating if the date is the first day of the month.
+
+        Returns
+        -------
+        is_month_start : Series of boolean
+
+        See Also
+        --------
+        is_month_end : Returns a boolean indicating if the date is the last day of the month.
+
+        Examples
+        --------
+        >>> import pandas as pd
+        >>> dates = pd.Series(pd.date_range("2018-02-27", periods = 3))
```

Comment:
PEP8 doesn't allow spaces around keyword parameters. i.e.: `periods=3`, instead of `periods = 3`

Reply:
`import pandas as pd` is not necessary.